LICENSE

```diff
@@ -1,21 +1,165 @@
-The MIT License
-
-Copyright (c) the original author or authors, https://github.com/ynop/audiomate
-
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
-
-The above copyright notice and this permission notice shall be included in
-all copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
-THE SOFTWARE.
+                   GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
Yes, it's because I'm developing all of this in my free time and want to make sure that everyone who uses and extends it also contributes back again, that it becomes an exchange of work and time.

What does speak against the license change? It shouldn't affect any users, only a very small amount of developers.

Reply:
License changes require extensive work. This cannot happen buried in some other pull request. For example, every contributor has to agree, otherwise we have to remove their contributions. This is not the place to discuss the license change. If you feel strongly about changing the license, open an issue and explain in detail why you think the MIT license isn't a good choice and why the license you're proposing solves that problem.

Reply:
Do you really need to ask every contributor? The MIT license is compatible to the LGPL so I think you would need to do this only when changing the other way.

Reply:
Yes, we'd have to. And clear it with legal and whatnot. Change is change.